src/common/Input/Input.tsx

```diff
@@ -0,0 +1,31 @@
+import './Input.css'
+import {
+    ChangeEvent,
+    useState
+} from 'react';
+
+// @ts-ignore
+export default function Input(props) {
```

Comment:
good practice to use props destructuring:
```
function Input({ value, placeholder, onChange, onKeyUp }) {
 ...
}
```
or
```
function Input(props) {
  const { value, placeholder, onChange, onKeyUp } = props
}
```

Reply:
Will add this update in Task 4 :)